gdpr.html

```diff
@@ -60,30 +61,30 @@ <h1 class="heading__title">Ochrana osobních údajů</h1>
+<p>2) Pokud budete sami pozitivně testováni na koronavirus, můžeme vás požádat o odeslání historie setkávání s dalšími telefony s nainstalovanou aplikací. Pokud nám tuto informaci dobrovolně předáte pomocí aplikace (konkrétně pomocí funkce “odeslat data”), pak tuto informaci využijeme k varování ohrožených osob a zjišťování dalších epidemiologických souvislostí. To znamená, že se snažíme zjistit, od koho jste se mohli nakazit a koho jste případně mohli nakazit. O odeslání vás požádáme v souvislosti s informací o vašem nakažení koronavirem, což je osobní údaj zvláštní kategorie.</p>
 <p>Jak dlouho s výše uvedenými údaji pracujeme? Informace o setkávání s jinými telefony s nainstalovanou aplikací se vymažou nejpozději po 30 dnech, ze svého telefonu je můžete vymazat kdykoliv. Když nám odešlete informace o setkávání, vyhodnotíme je a smažeme do 12 hodin od jejich odeslání z vašeho telefonu. Vaše telefonní číslo smažeme nejpozději do šesti měsíců od dokončení instalace aplikace, případně dříve, pokud jej již nebudeme potřebovat nebo pokud odvoláte svůj souhlas s jeho zpracováním.</p>
-<p>S výše uvedenými údaji pracujeme na základě vašeho souhlasu, jde tedy o zpracování podle čl. 6 bod 1 písm.a) a čl. 9 bod 2 písm. a) GDPR – příslušnou volbou v aplikaci souhlasíte se zpracováním osobních údajů výše uvedeným způsobem za účelem ochrany veřejného zdraví, konkrétně za účelem epidemiologického šetření. Váš souhlas je dobrovolný. Bez něj ale nemůžeme vaše údaje použít k výše uvedeným účelům a aplikace bez vašeho souhlasu tedy nemůže fungovat. Souhlas můžete kdykoliv odvolat vaplikaci volbou “zrušit registraci” (tím se vymaže z databáze vaše telefonní číslo) a odinstalováním aplikace (pak už nebudou ostatní telefony s aplikací zjišťovat, že vás jejich uživatelé potkali). Pokud už jsme data použili pro epidemiologické šetření, nelze tato zjištění technicky vzít zpět.</p>
+<p>S výše uvedenými údaji pracujeme na základě vašeho souhlasu, jde tedy o zpracování podle čl. 6 bod 1 písm.a) a čl. 9 bod 2 písm. a) GDPR – příslušnou volbou v aplikaci souhlasíte se zpracováním osobních údajů výše uvedeným způsobem za účelem ochrany veřejného zdraví, konkrétně za účelem epidemiologického šetření. Váš souhlas je dobrovolný. Bez něj ale nemůžeme vaše údaje použít k výše uvedeným účelům a aplikace bez vašeho souhlasu tedy nemůže fungovat. Souhlas můžete kdykoliv odvolat v aplikaci volbou “zrušit registraci” (tím se vymaže z databáze vaše telefonní číslo) a odinstalováním aplikace (pak už nebudou ostatní telefony s aplikací zjišťovat, že vás jejich uživatelé potkali). Pokud už jsme data použili pro epidemiologické šetření, nelze tato zjištění technicky vzít zpět.</p>
```

Comment:
Změnil bych na `Váš souhlas je dobrovolný, bez něj ale nemůžeme` - teď je to jako dvě věty a je to divný...